src/services/iasql.ts

```diff
@@ -713,11 +713,12 @@ export async function install(moduleList: string[], dbAlias: string, user: any,
     }`);
   }
   // Check to make sure that all dependent modules are in the list
-  const missingDeps = mods.map((m: Modules.ModuleInterface) => m.dependencies.find(d => !moduleList.includes(d)));
```

Comment:
`find` only returns the first value that matches the predicate so this wasn't returning a full list

Reply:
hmm curious how it was working or why it was done with the `find`?

Reply:
it was always returning `aws_account` which was the first match and that was then filtered so it was a always a noop